nodeJS/final_project/project_odin_book.md

```diff
@@ -1,42 +1,50 @@
+1. There should be an index page for posts, which shows all the recent posts from the current user and users they are following.
+1. Users can create a profile with a profile picture. You may be able to get the profile picture when users sign in using Omniauth. If this isn't the case you can use [Gravatar](https://www.gravatar.com/) to generate them
+1. A user's profile page should contain their profile information, profile photo, and posts.
+1. There should be an index page for users, which shows all users and buttons for sending follow requests to users the user is not already following or have a pending request.
+1. Deploy your app to a hosting provider of your choice!
 
 #### Getting started
 
-1. Think through the data architecture required to make this work.  There are lots of models and the relationship between them is more complicated than anything you've done before.  How are you going to model a user's list of friends and friend requests? Posts should be able to have likes and comments associated with them, how are you going to model that? Take some time to plan your approach before diving in.
-2. Start your app however you like, using the express-generator or from scratch.
-3. Work your way down the list above!  Each step will involve a new challenge, but you've got the tools.
-4. You can populate data like users and posts with fake data using the [Faker](https://github.com/faker-js/faker) module from npm.  To accomplish this create a new JavaScript file named `seeds.js` which imports your mongoose models and uses the faker module to generate and save a bunch of new users.
+1. Think about what you need to do. It's really helpful to write your plan down on paper or a whiteboard ahead of time! A few hours of thought now will save you days of coding. Try to lay it ALL out. An important part of planning is **scope**. You obviously can't build the entire website (which presumably took a full team of engineers years to produce), so you'll need to identify the site's core functionality and the "nice-to-have" stuff. **Make sure you finish the core functionality BEFORE working on the rest.** If you try to do everything at once, you'll get lost and frustrated. Trust us. Everything takes longer than you expect.
+1. Think through the data architecture required to make this work. There are lots of models and the relationship between them is more complicated than anything you've done before. Take some time to plan your approach before diving in.
+1. Start your app however you like, using the express-generator or from scratch.
+1. Work your way down the list above! Each step will involve a new challenge, but you've got the tools.
+1. You can populate data like users and posts with fake data using the [Faker](https://github.com/faker-js/faker) module from npm. To accomplish this create a new JavaScript file named `seeds.js` which imports your Mongoose models and uses the faker module to generate and save a bunch of new users.
```

Comment:
Since we are rewriting this lesson, what about moving the getting started section prior to assignment section (with minor wording tweaks). I feel this section is misplaced. Thoughts?